### PR TITLE
Add Material Theme

### PR DESCRIPTION
**Pull request by dantehemerson**
Themes included:

* Material Theme Light
* Material Theme Oceanic
* Material Theme Dark

### CODE REVIEW

File: themes/prism-material-dark.css [hunk 1]
```diff
@@ -0,0 +1,220 @@
+code[class*="language-"],
+pre[class*="language-"] {
+  text-align: left;
+  white-space: pre;
+  word-spacing: normal;
+  word-break: normal;
+  word-wrap: normal;
+  hyphens: none;
+  color: inherit;
```
Comment (RunDevelopment) [inline themes/prism-material-dark.css]: I find this very strange. You use `inherit` here but for both `:not(pre) > code[class*="language-"]` and `pre[class*="language-"]`, you use `#eee`. Why not set it here?

Same for `background`.

Also, same for the two other themes as well.

File: themes/prism-material-dark.css [hunk 2]
```diff
@@ -0,0 +1,220 @@
+code[class*="language-"],
+pre[class*="language-"] {
+  text-align: left;
```
Comment (RunDevelopment) [inline themes/prism-material-dark.css]: We use one tab as indentation.

File: themes/prism-material-dark.css [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+  color: #eee;
+  background: #2f2f2f;
+  border-radius: 0;
+  margin: 0.5em 0;
+  padding: 1.25em 1em;
+}
+
+pre[class*="language-"].language-css > code,
+pre[class*="language-"].language-sass > code,
+pre[class*="language-"].language-scss > code {
```
Comment (mAAdhaTTah) [inline themes/prism-material-dark.css]: You don't need the `[class*="language-"]` if you're specifying `.language-css` etc. right after

Comment (mAAdhaTTah) [reply themes/prism-material-dark.css]: Same issue on all themes.

File: themes/prism-material-dark.css (hunk 1 above)
Comment (dantehemerson) [reply themes/prism-material-dark.css]: You're right, I'll modify that.

File: themes/prism-material-dark.css (hunk 3 above)
Comment (dantehemerson) [reply themes/prism-material-dark.css]: You're right, I fixed this.

**Comment by dantehemerson** [comment]
@mAAdhaTTah @RunDevelopment  First of all, sorry for the delay and thanks for the review.
I changed the styles. I hope these changes solve the reviews.

**Pull request merged**